.github/workflows/js_api.yml

```diff
@@ -101,10 +103,19 @@ jobs:
               with:
                   node-version: 14
 
+            # needed because of # https://github.com/rustwasm/wasm-bindgen/issues/2261
+            - name: Install geckodriver
+              if: matrix.conf == 'firefox'
+              working-directory: ${{ env.RAV1E_JS_DIR }}
+              run: |
+                  wget https://github.com/mozilla/geckodriver/releases/download/v0.26.0/geckodriver-v0.26.0-linux64.tar.gz
```

Comment:
Add the URL to a `LINK` env variable like [here](https://github.com/xiph/rav1e/blob/master/.github/workflows/rav1e.yml#L106)

Reply:
Okay :+1: 

That's actually way cleaner